history/ReadMe.md

```diff
@@ -48,4 +48,4 @@ Some other improvements that could be conducted:
 * configure the [extraction.properties](extraction.properties) file
 * and run  ```../run run extraction.properties```
 
-* Test it with : mvn test (need to have a containing file frwiki-[YYYYMMDD]-download-complete empty flag file into the base-dir defined into the extraction-properties file )
\ No newline at end of file
+* Test it with `mvn test` (need to have a containing file, `frwiki-[YYYYMMDD]-download-complete` empty flag file into the `base-dir` defined into the `extraction-properties` file)
```

Comment:
This could use some rewording, as it's not clear what's meant by what's here. (I made some tweaks that may _help_ with clarity, but they're insufficient on their own.)